src/lib/notifications.ts

```
@@ -0,0 +1,60 @@
+import { useState, useEffect, useRef } from 'react';
```

ESLint

[error] 'useState' is defined but never used.

(@typescript-eslint/no-unused-vars)